zaza/openstack/charm_tests/glance_simplestreams_sync/tests.py

```diff
@@ -125,3 +125,24 @@ def _check_local_product_streams(expected_images):
         _check_local_product_streams(expected_images)
 
         logging.debug("Local product stream successful")
+
+
+class GlanceSimpleStreamsSyncWithPropertiesTest(GlanceSimpleStreamsSyncTest):
+
+    @classmethod
+    def setUpClass(cls):
+        # `setup.py:set_latest_property_config()` is required by this test and
+        # it is called by
+        # charm-glance-simplestreams-sync/tests/tests.yaml:configure phase
+        super(GlanceSimpleStreamsSyncWithPropertiesTest, cls).setUpClass()
```

Comment:
Sorry, to be a bit pedantic, but this override on `setUpClass` isn't needed any more.  The comment re: set_latest_property_config() is useful, and should be retained in the class's docstring.